contextConsumption/query_entities_test.js

```diff
@@ -187,7 +188,7 @@ describe('Query Entity. JSON. Default @context', () => {
   it('query by condition over object', async function() {
     const queryParams = {
       type: entity.type,
-      q: 'R100=="urn:ngsi-ld:T2:6789"'
+      q: 'R100==urn:ngsi-ld:T2:6789'
```

Comment:
This is actually not a bug in the test suite.
The original q follows the ngsi-ld spec, while the modification follows APIv2.
Right now, orionld uses the old APIv2 q. I am working on the new "ngsi-ld q" right now.
Will be ready next week, if not this week :)

Might not be a good idea to change this

Reply:
Change undone in b0799dd